src/vscode-bicep-ui/packages/components/src/pan-zoom/atoms.ts

```diff
@@ -0,0 +1,18 @@
+import { atom } from "jotai";
```

Comment:
Is atoms.ts just used to provide access to atom from jotai?

Reply:
Yes.

Reply:
Oh, and to provide [isolation](https://jotai.org/docs/extensions/scope#createisolation).